charts/argo-rollouts/templates/controller/clusterrole.yaml

```diff
@@ -267,5 +267,14 @@ rules:
     - watch
     - update
 {{- end }}
+{{- if .Values.providerRBAC.providers.glooPlatform }}
+  # Access needed when using the Gloo Platform provider
+- apiGroups:
+    - networking.gloo.solo.io
+  resources:
+    - routetables
+  verbs:
+    - '*'
```

Comment:
Hi @bgottfried91 , can you please give us the doc what you refer for this config?

Reply:
Hi @yu-croco - this resource is used for traffic delegation via RouteTables for Gloo Edge
Ref: https://docs.solo.io/gloo-edge/latest/introduction/traffic_management/

Reply:
The plugin is mentioned in values.yaml 
Ref: https://github.com/argoproj-labs/rollouts-plugin-trafficrouter-glooplatform/blob/main/README.md

Reply:
Hi @pdrastil , thank you for the info ! 👍

Reply:
@yu-croco FYI - it seems that Gloo is adopting new Kubernetes Gateway API
Ref: https://github.com/solo-io/gloo/tree/v2.0.x

Reply:
Just as a FYI, this addition is to support the plugin for Gloo Platform. There's a separate plugin for Gloo Edge, they're different APIs. We'll probably submit an update to support adding the RBAC for that plugin soon!

This is the overview page for Gloo Platform's traffic management (https://docs.solo.io/gloo-mesh-enterprise/latest/concepts/traffic-management/) and the API reference for it (https://docs.solo.io/gloo-mesh-enterprise/latest/reference/api/route_table/)

Petr's correct that GE builds on the Gateway API now though!

Reply:
Thank you for the info :)